### PR TITLE
feature/IIA-751 mock up reasoner menu

### PR DESCRIPTION
**Pull request by dholubek**
This is the UI mock up of a reasoner menu from the Concept window in the Journal view with no functionality behind the menu options as of yet.  This converts this reasoner from a toggle button to a regular button

image

### CODE REVIEW

File: kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/details/concept-details.fxml [hunk 1]
```diff
@@ -72,16 +72,20 @@
...
                            </graphic>
                         </Button>
-                        <ToggleButton fx:id="reasonerToggleButton" mnemonicParsing="false" onAction="#openReasonerSlideout" styleClass="reasoner" text="ToggleButton">
+                        <Button fx:id="reasonerToggleButton" layoutX="96.0" layoutY="16.0" mnemonicParsing="false" onAction="#popupAddContextMenu" styleClass="reasoner" text="Button">
+                           <contextMenu>
+                              <ContextMenu fx:id="reasonerContextMenu" opacity="0.99" styleClass="kview-context-menu">
+                                 <items>
+                                    <MenuItem mnemonicParsing="false" text="Run full reasoner" />
+                                    <MenuItem mnemonicParsing="false" text="Run incremental reasoner" />
+                                    <MenuItem mnemonicParsing="false" text="Redo navigation" />
```
Comment (carldea) [inline kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/details/concept-details.fxml:81]: (Optional) Can you add functions that get called for each menu item? 
My reasoning: Whoever is assigned the implementation ticket could focus on the controller class.

Comment (dholubek) [reply kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/details/concept-details.fxml:81]: yes... will do...

Comment (dholubek) [reply kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/details/concept-details.fxml]: updated

**Review by carldea — APPROVED** [review]
Looks good!

**Pull request merged**